hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerImpl.java

```diff
@@ -638,8 +633,6 @@ public void testInvalidPrefixAcl() throws IOException {
         .build();
 
     // add acl with invalid prefix name
```

Comment:
Hi @adoroszlai @hemantk-12 can you help review this PR?
Especially the following lines in `testInvalidPrefixAcl()`.